include/revng/Pipeline/LLVMContainer.h

```diff
@@ -196,21 +202,25 @@ class LLVMContainerBase
 
     llvm::Linker TheLinker(*Composite);
 
-    bool Result = TheLinker.linkInModule(std::move(Module),
-                                         llvm::Linker::OverrideFromSrc);
+    bool Failure = TheLinker.linkInModule(std::move(Module),
+                                          llvm::Linker::OverrideFromSrc);
 
-    Result = Result
-             and TheLinker.linkInModule(std::move(ToMerge.Module),
+    Failure = Failure
+              or TheLinker.linkInModule(std::move(ToMerge.Module),
                                         llvm::Linker::OverrideFromSrc);
```

Comment:
What was the bug?
What's the fix?

Try to be a bit more verbose in commit messages `:P`

Reply:
done!
in this case it was that linkInModule returned true on failure for some reason.